wheel/generate_type_stubs.py

```diff
@@ -409,3 +412,9 @@ def derive_child_pk_unhardened(pk: G1Element, index: int) -> G1Element: ...
 
     for c in classes:
         print_class(f, c[0], c[1], extra_members.get(c[0]))
+
+    for c in raw_classes:
+        f.write(f"\nclass {c[0]}:\n")
+        for m in c[1]:
+            f.write(f"    {m}\n")
```

Comment:
by treating this class differently, you don't expose the `replace()` function. I believe we use this function on `ConsensusConstants` in `chia-blockchain`. You also don't expose the streamable interface to python.

Is there a good reason to not parse `ConsensusConstants` into `classes` instead of `raw_classes`?